packaging/functions.sh

```diff
@@ -344,10 +344,10 @@ install_linux_appdata() {
 			fi
 		fi
 
-		install -d "${SHARE_PATH}/appdata"
+		install -d "${SHARE_PATH}/metainfo"
 
 		sed "s/{MODID}/${MOD_ID}/g" "${SRC_PATH}/packaging/linux/openra.appdata.xml.in" | sed "s/{MOD_NAME}/${MOD_NAME}/g" | sed "s/{SCREENSHOT_RA}/${SCREENSHOT_RA}/g" | sed "s/{SCREENSHOT_CNC}/${SCREENSHOT_CNC}/g" | sed "s/{SCREENSHOT_D2K}/${SCREENSHOT_D2K}/g"> "${SRC_PATH}/packaging/linux/openra-${MOD_ID}.appdata.xml"
-		install -m644 "${SRC_PATH}/packaging/linux/openra-${MOD_ID}.appdata.xml" "${SHARE_PATH}/appdata"
+		install -m644 "${SRC_PATH}/packaging/linux/openra-${MOD_ID}.appdata.xml" "${SHARE_PATH}/metainfo"
 		rm "${SRC_PATH}/packaging/linux/openra-${MOD_ID}.appdata.xml"
```

Comment:
While [the spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-location) says that appdata.xml remains acceptable, IMO we should change this to metainfo.xml for best practice.

Reply:
Oh, I didn't know that. Most applications use `metainfo` path, but `.appdata.xml` as filename.